google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -1316,17 +1326,15 @@ public Database getDatabase(String databaseName) throws SpannerException {
   }
 
   @Override
-  public List<String> getDatabaseDdl(String databaseName) throws SpannerException {
+  public GetDatabaseDdlResponse getDatabaseDdl(String databaseName) throws SpannerException {
```

Comment:
I don't expect any customer to be using this API directly but if someone is using it, it will be breaking change.

Reply:
I guess `GapicSpannerRpc.java` class is marked as `@InternalApi`. This means the code is subject to change and customers can use it at their own risk. I have added necessary changes to clirr file to mark this change.
@rajatbhatta @gauravpurohit06  correct me if I am missing something here.

Reply:
Makes sense to me.